editors/code/package.json

```diff
@@ -416,7 +416,7 @@
                 },
                 "rust-analyzer.cargo.runBuildScripts": {
                     "markdownDescription": "Run build scripts (`build.rs`) for more precise code analysis.",
-                    "default": false,
+                    "default": true,
```

Comment:
I was wondering why OpenGL function calls from [gl-rs](https://github.com/brendanzab/gl-rs) were now finally being highlighted after the latest update. Now I know why :)